File: C/Fizz_Buzz_jayaganesh.c
#include <stdio.h>

int main()
{
    int i;
    printf("1, 2,");
    for(i=3;i<100;i++){
        if(i%3==0 && i%5==0){
            printf(" FizzBUzz,");
        }
        else if(i%3==0 && i%5!=0){
            printf(" Fizz,");
        }
        else if(i%5==0 && i%3!=0){
            printf(" Buzz,");
        }
        else{
            printf(" %d,",i);
        }
    }
    printf(" %d",100);

    return 0;
}

File: Python/fizzbuzz_jayaganesh.py
for i in range(1,100):
    if i % 15 == 0:
        print("FizzBuzz,", end=' ')
    elif i%3==0:
        print("Fizz,", end=" ")
    elif i%5==0:
        print("Buzz,",end=' ')
    else:
        print(i,",",end=" ")
print(100)